src/AppInstallerCLIE2ETests/WinGetUtil/WinGetUtilWrapper.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AppInstallerCLIE2ETests.WinGetUtil
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// Wrapper class for WinGetUtil exports.
+    /// For more details about methods in this class visit WinGetUtil.h file.
+    /// </summary>
+    internal class WinGetUtilWrapper
+    {
+        private const string DllName = @"WinGetUtil.dll";
+
+        [Flags]
+        public enum CreateManifestOption
```

Comment:
>CreateManifestOption [](http://example.com/codeflow?start=20&length=20)

I think at least 1 failure case or SchemaAndSemanticValidation should be tested to trigger the manifest validation code path.

Reply:
Also, even if we are not using them yet, we should have the complete copy of the enum values for completeness